ui/imports/utils/Constants.qml

```diff
@@ -53,6 +53,7 @@ QtObject {
         readonly property int profile: 4
         readonly property int timeline: 5
         readonly property int communityChat: 6
+        readonly property int category: 9999
```

Comment:
@richard-ramos I think this is not needed in this context, if you agree? It is about `chatType` and possible options a chat may have, there is no need for category here, lets imply that if it's a category it will have `unknown` chat type. Of course if you agree?

Reply:
Agreed. I removed the fake chat type, and now determine if it's a category or not comparing against the `unknown` chat type